client/src/components/ui/itemrow/SendRow.svelte

```diff
@@ -13,8 +17,25 @@
     export let category: Category['name'];
     export let title: Document['title'];
     export let target: Snapshot['target'];
+    export let creation: Snapshot['creation'];
+
+    function redirectHandler() {
+        window.location.href = `/track?id=${doc}`;
+    }
+
+    const targetName = target ? $allOffices[target] : '';
+    const docDisplay = `${doc.slice(0,5)}...${doc.slice(-5)}`;
```

Comment:
Prefer the CSS property [`text-overflow`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow).